custom/abdm/migrations/0001_squashed_0002_auto_20220926_0547.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-09-26 05:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('abdm', '0001_initial'), ('abdm', '0002_auto_20220926_0547')]
```

Comment:
migration `0001_initial` should be present as well?

Reply:
Will remove this before merging, have to make sure that staging is not affected by orphan migration with no reference to deleted one.